apps/google-analytics-4/frontend/src/components/config-screen/WarningDisplay/constants/warningMessages.ts

```diff
@@ -13,6 +14,13 @@ export const getSlugFieldDeletedMsg = (contentTypeId: string, slugField: string)
   Please select a new slug field in order for the app to render correctly in the sidebar. `;
 };
 
+// Property selection warning and error messages
+export const NO_PROPERTIES = `There are no properties to display.`;
```

Comment:
Copy suggestions:

> The service account this app is linked to has not been given access to any analytics properties.

or something similar? this suggestion tries to describe the problem and suggest how they could solve it.

Reply:
This component displays the Note instead of the dropdown when there are no account summaries; it is not aware of the API error state. So I purposely kept the content generic here thinking that the user would be able to look in the API access section for more descriptive info. If I update the content to something more descriptive, then it may not apply to a a general API error as the first screenshot shows. So I think I'd prefer to keep this more generic. What do you think?